exocom-server/features/support/cli-world.ls

```diff
@@ -27,7 +27,8 @@ CliWorld = !->
 
   @create-mock-service-at-port = ({name, port, namespace}, done) ->
     (@service-mocks or= {})[name] = new MockService {port, name, namespace}
-    wait 200, done
+    @service-mocks[name].connect {}, ->
+      wait 200, done
```

Comment:
You're working on a PR for exocom-server to reply with a message once a service has registered itself right? Does that mean this `wait` statement will instead be for that reply message in the future?